src/views/Hero/index.view.tsx

```diff
@@ -37,7 +37,7 @@ const HeroView: React.FC<HeroProps> = ({
         <div className="HeroView__textContainer">
           {title.map((item) => (
             <h1 className={`HeroView__titleText${item.style}`} key={item.text}>
-              {item.text}
+              {item.text} /
```

Comment:
This adds a slash to the secondary header too. Is it intended?

Reply:
Ohh, yeah. I didn't see the `map()` - I'll undo this.